execution_engine/src/runtime/mod.rs

```diff
@@ -1081,6 +1081,11 @@ where
             }
             (EntryPointType::Session, EntryPointType::Contract)
             | (EntryPointType::Contract, EntryPointType::Contract) => Ok(contract_hash.into()),
+
```

Comment:
I think we need to add another case here for `Installer` calling into `Session`

Reply:
Good catch; fixed